adam-core/src/test/scala/org/bdgenomics/adam/rdd/features/FeatureRDDFunctionsSuite.scala

```diff
@@ -0,0 +1,256 @@
+/**
+ * Licensed to Big Data Genomics (BDG) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The BDG licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.bdgenomics.adam.rdd.features
+
+import java.io.File
+import org.apache.spark.rdd.RDD
+import org.bdgenomics.adam.rdd.ADAMContext._
+import org.bdgenomics.adam.rdd.features.FeatureRDDFunctionsSuiteHelper._
+import org.bdgenomics.adam.util.ADAMFunSuite
+import org.bdgenomics.formats.avro._
+import org.scalactic.{ Equivalence, TypeCheckedTripleEquals }
+
+object FeatureRDDFunctionsSuiteHelper {
+  def sortByExon(features: RDD[Feature]): RDD[Feature] = {
+    def exon(f: Feature) =
+      (
+        f.getContig.getContigName,
+        f.getStart,
+        f.getEnd,
+        f.getFeatureType,
+        f.getAttributes.getOrElse("transcript_id", "~"),
+        f.getAttributes.getOrElse("exon_number", Int.MaxValue.toString).toInt
+      )
+    features.coalesce(1).sortBy(exon)
+  }
+
+  def sortByReference(features: RDD[Feature]): RDD[Feature] = {
+    def reference(f: Feature) =
+      (
+        f.getContig.getContigName,
+        f.getStart,
+        f.getEnd
+      )
+    features.coalesce(1).sortBy(reference)
+  }
+}
+
+class FeatureRDDFunctionsSuiteHelper extends Serializable {
+  // empty
```

Comment:
You can just add that to the singleton object, no? E.g., `object FeatureRDDFunctionsSuiteHelper extends Serializable`